api/envoy/admin/v2alpha/listeners.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package envoy.admin.v2alpha;
+
+option java_outer_classname = "ListenersProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.admin.v2alpha";
+
+import "envoy/api/v2/core/address.proto";
+
+// [#protodoc-title: Listeners]
+
+// Admin endpoint uses this wrapper for `/listeners` to display listener status information.
+// See :ref:`/listeners <operations_admin_interface_listeners>` for more information.
+message Listeners {
+  // List of listener statuses.
+  repeated ListenerStatus listener_statuses = 1;
+}
+
+// Details an individual listener's current status.
+message ListenerStatus {
```

Comment:
nit: Should we just call it `Listener` instead of `ListenerStatus`?

Reply:
I followed the naming scheme of clusters.proto since both `/clusters` and `/listeners` serve the same purpose of surfacing runtime information (rather than config info).

Reply:
OK. Makes sense.